tools/packaging/kata-deploy/Dockerfile

```diff
@@ -26,4 +17,6 @@ echo "gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packa
 yum -y install kubectl && \
 yum clean all
 
+RUN mkdir -p ${DESTINATION}
+ADD ${KATA_ARTIFACTS} ${DESTINATION}/
```

Comment:
CI tools dose not like this:

```
tools/packaging/kata-deploy/Dockerfile:21 DL3020 error: Use COPY instead of ADD for files and folders
[1410](https://github.com/kata-containers/kata-containers/actions/runs/3345467724/jobs/5566985626#step:11:1411)
ERROR: failed to check Dockerfile 'tools/packaging/kata-deploy/Dockerfile'
```

Not sure why this policy is enforced. Using ADD instead here, docker will automatically extract files into target, hence reduce the size of the images with about 100MB.

Reply:
This seems to be an static check warning that we could ignore.

Reply:
@wainersm  may you please help us to get this move on.